modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -145,7 +145,7 @@ const EditorManager: EditorManager = {
    * @property pageUid
    * @type String
    */
-  pageUid: null as any,
```

Comment:
I have changed the `pageUid` to be created here rather than on `setup`. It is technically possible for `EditorManager.setup` to be called more than once and if it Is called again, it throws an error as `pageUid` is already locked from the previous invocation of `EditorManager.setup`